server/server.go

```diff
@@ -821,7 +832,9 @@ func (s *StanServer) createNatsClientConn(name string, sOpts *Options, nOpts *se
 		ncOpts.ReconnectBufSize = 128
 	}
 
-	Tracef(" NATS conn opts: %v", ncOpts)
+	if s.trace {
```

Comment:
But there are many places in the code where I do that. It happens that you saw this one because of the way the code review is.